internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -214,7 +221,14 @@ func (handler StorageHandler) newCommandValue(resourceName string, reading inter
 		if !ok {
 			return nil, fmt.Errorf(castError, resourceName, "not []byte")
 		}
-		result, errn = models.NewCommandValue(resourceName, timestamp, val, valueType)
+		if resourceName == "jpeg" || resourceName == "png" {
+			Base64Val := convertToBase64(val)
+			valueType = models.String
+			resourceName = "string"
```

Comment:
I think we don't need to change the resourceName.

Reply:
I tested it, It works. Thanks !
![Name](https://user-images.githubusercontent.com/63772873/118128989-3cabb480-b419-11eb-92ab-cd08d893aada.png)